util/tracing/opt_trace.go

```diff
@@ -80,4 +82,5 @@ type LogicalRuleOptimizeTraceStep struct {
 	Reason string `json:"reason"`
 	ID     int    `json:"id"`
 	TP     string `json:"type"`
+	Index  int    `json:"Index"`
```

Comment:
Should we unify the upper and lower case in the JSON?

Reply:
fixed and updated.